### ISSUE TITLE
Add support for external flowmaps

### ISSUE THREAD

**Issue by arunoda** [GameDev4K/unreal-openland#39]
It'd be interesting use these flow maps to do various things. Coupling that smart mask is a good option too.

Some videos: https://www.youtube.com/watch?v=GztL0Oc8emE

**Comment by fortek12** [comment]
Hi Arunoda! Is there a status update on this item? This would be a very common use case (like importing a detailed local terrain mesh around a building from Civil 3D or other external software), and I would love to be able to use OpenLand to auto-material those static meshes, especially since OpenLand has one-click support for RVT and Virtual HeightField Mesh.

**Comment by arunoda** [comment]
Yeah! There are customers still doing this with our [Modular dev setup](https://www.notion.so/gamedev4k/OpenLand-Documentation-2268081d3b8e4a49a0d824a7ab0b7b44#06cda09bb5644a91b0bf60fce33bbd0a). But, I will add an example master material for ease of use.
